### ISSUE TITLE
minify issue for 2.12.16 - 2.12.7

### ISSUE THREAD

**Issue by lujiacn** [tdewolff/minify#591]
during minify my js file `minify -r -b -o ../public/bundle_dist/bundle12.js  --match=\.js ../public/bundle_dist/bundle12` , encounter the error as below:
```
: unexpected { in expression on line 26 and column 5
   26: [· ]{`+Math.min.apply(Math,w)+"}","g");p=p.map(function(_...
           ^
```
which only happend for version 2.12.16, 2.12.17. 

Using 2.12.15, no issue.

**Comment by tdewolff** [comment]
Could you share the input file that causes the problem?

**Comment by lujiacn** [comment]
the issue can be closed, works fine with 2.12.18 :p
